src/components/home/home.styles.js

```diff
@@ -12,7 +12,7 @@ const useHomeStyles = createUseStyles(theme => ({
     [theme.breakpoints.sm]: {
       paddingTop: '320px'
     },
-    backgroundImage: 'url(./home-background.svg)',
+    backgroundImage: 'url(./home-background.png)',
```

Comment:
Mmmm I'm not sure how a `png` will scale in bigger/smaller screens.. I think that we should talk with Chami to have an `.svg` here

Reply:
It was Chami who said to use `.png`, since she is having some problems with `.svg` shadows.